### DISCUSSION TITLE
Auto-fan_action mode similar to auto-stop

### DISCUSSION THREAD

**Discussion by 2x4ever** [jmcollin78/versatile_thermostat#679] [Ideas]
It would be very cool to have an autofan_action mode that would repeat the auto-stop mode in behavior.

I.e., when there is no longer a need for cooling or heating, switch the air conditioner to ventilation mode. 
This is necessary for better mixing of the air in the room. 
In addition, the air conditioner has a filter and an ionizer, which I want to be always turned on.

**Comment by gioele-antoci** [comment]
I would really like this feature because my heat pump won't stop heating for 20 mins even if the underlying target temperature is lower than the temperature detected (by the underlying itself or by the versatile)

**Comment by d96moe** [comment]
I was just about to ask for the same! As my HVACs still uses power even when set to off (like 300W when it's around 0 degC outside to keep the outside module in working temperature), keeping the inside fan running would help pushing the air around in the house to keep a more even overall house temperature.

**Comment by d96moe** [comment]
Question: Should I create a feature request issue out of this, or is this done when an awesome dev starts on a PR?

**Reply by jmcollin78** [reply]
You could create or transform this discussion into an issue of type Feature Request.
